tests/unit_tests/tasks/test_cron_util.py

```diff
@@ -14,11 +14,9 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from datetime import datetime
 
 import pytest
-import pytz
-from dateutil import parser
-from freezegun import freeze_time
```

Comment:
Is this used elsewhere? If not we should remove the `freezegun` library.

Reply:
Yes, it's still used elsewhere

Reply:
I *love* freezegun, hands off @john-bodley! 😄